test/unit/extract-runnable-items.test.js

```diff
@@ -487,6 +487,40 @@ describe('extractRunnableItems', function () {
                 }
             );
         });
+
+        describe('Preserve Order', function () {
+            it('should work correctly when not specified', function (done) {
```

Comment:
Test description doesn’t tell what it's asserting.

e.g, `should follow the collection order when unset`.

Reply:
@codenirvana 
I'll do the needful.